identity/src/keypair.rs

```diff
@@ -227,15 +242,16 @@ impl Keypair {
     }
 
     /// Encode a private key as protobuf structure.
-    #[cfg_attr(
-        not(feature = "ed25519"),
-        allow(unreachable_code, unused_variables, unused_mut)
-    )]
-    pub fn to_protobuf_encoding(&self) -> Result<Vec<u8>, DecodingError> {
-        use quick_protobuf::MessageWrite;
+    #[deprecated(since = "0.2.0", note = "Renamed to `encode_protobuf`")]
+    pub fn to_protobuf_encoding(&self) -> Vec<u8> {
```

Comment:
This function needs to retain its old function signature to actually be non-breaking.